config/rbac/role.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: manager-role
```

Comment:
Needs a more specific name

Reply:
We won't use this, these are created by kube-builder. Will nuke those files.